.github/workflows/publish-engines.yml

```diff
@@ -38,14 +38,23 @@ jobs:
           GITHUB_EVENT_CLIENT_PAYLOAD: ${{ toJson(github.event.client_payload) }}
 
       - name: Workflow dispatch to prisma/prisma for version update
-        if: ${{ steps.publish_script.outputs.new_prisma_version }}
+        # Only automatically trigger update for `latest`!
+        if: steps.publish_script.outputs.npm_dist_tag == 'latest' 
         uses: benc-uk/workflow-dispatch@v1
         with:
           workflow: Update Engines Version
           repo: prisma/prisma
           token: ${{ secrets.BOT_TOKEN }}
           inputs: '{ "version": "${{ steps.publish_script.outputs.new_prisma_version }}" }'
 
+      - name: Workflow dispatch to prisma/prisma-fmt-wasm for version update
+        uses: benc-uk/workflow-dispatch@v1
+        with:
+          workflow: new-engines-version
+          repo: prisma/prisma-fmt-wasm
+          token: ${{ secrets.BOT_TOKEN }}
+          inputs: '{ "engineHash": "${{ github.event.client_payload.commit }}", "enginesWrapperVersion": "${{ steps.publish_script.outputs.new_prisma_version }}", "npmDistTag": "${{ steps.publish_script.outputs.npm_dist_tag }}" }'
```

Comment:
new dispatch to `prisma-fmt-wasm` repo on the `new-engines-version` workflow, with 3 input values: `engineHash`, `enginesWrapperVersion`, and `npmDistTag`.